src/helpers/utils.ts

```diff
@@ -11,3 +14,11 @@ export const copyToClipboard = (str: string) => {
 export const timeout = (ms: number) => {
   return new Promise((resolve) => setTimeout(resolve, ms))
 }
+
+export const versionCompare = () => {
```

Comment:
This helper needs some tests. Please test multiple test cases.

I think the name can be improved. `versionCompare` tells me that it compares versions but it doesn't say which versions. I'd assume that you have to pass them as arguments.